docs/notebooks/deep_gaussian_processes.pct.py

```diff
@@ -0,0 +1,355 @@
+# %% [markdown]
+# # Using deep Gaussian processes with GPflux for Bayesian optimization.
+
+# %%
+import numpy as np
+import tensorflow as tf
+
+np.random.seed(1794)
+tf.random.set_seed(1794)
+tf.keras.backend.set_floatx("float64")
```

Comment:
we need to add a note, why this float treatment - to consistently stress this issue, and arguably here it will be most visible

Reply:
Done